account-kit/react/src/components/auth/card/error/connection-error.tsx

```diff
@@ -22,13 +19,18 @@ export const ConnectionError = ({
   handleUseAnotherMethod,
 }: ConnectionErrorProps) => {
   const getHeadingText = useMemo(() => {
+    const walletName =
```

Comment:
if your checking for validity in line 33 anyway just keep it undefined then.

Reply:
Yeah, this is just a preference, since the expected type of `walletName` is a string, resolving it to an empty string which is also falsy preserves the behaviour, but maintains the type. Happy to change this to be undefined though.

Reply:
Let's keep it undefined so simplify if it's used later on, so we can get compile time errors for lack of specification.